src/components/common/VideoCardMenu.vue

```diff
@@ -115,7 +116,7 @@
       {{ $t("component.videoCard.uploadScript") }}
     </v-list-item>
     <v-list-item
-      v-if="isLive"
+      v-if="video.type !== 'placeholder'"
```

Comment:
Popout should never show up for past chat because it doesn't work even with the extension.